src/components/Trade/Trade.tsx

```diff
@@ -0,0 +1,89 @@
+import { CSSProperties } from 'react';
+
+import cn from 'classnames';
+import { useTheme } from 'ui';
+
+import { useAppSelector } from 'hooks';
+
+import Breadcrumb from '../Breadcrumb';
+import TradeFormClosed from '../TradeForm/TradeFormClosed';
+import TradeFormInput from '../TradeForm/TradeFormInput';
+import { views } from './Trade.config';
+import styles from './Trade.module.scss';
+import type { View } from './Trade.types';
+import TradeActions from './TradeActions';
+import TradeDetails from './TradeDetails';
+import TradePredictions from './TradePredictions';
+
+type TradeProps = {
+  view?: View;
+};
+
+function Trade({ view = 'default' }: TradeProps) {
+  const theme = useTheme();
+  const market = useAppSelector(state => state.market.market);
+  const colors = useAppSelector(state => state.ui.market.colors);
```

Comment:
Houve um enhancement na cor dos markets, pois foi preciso adicionar as cores dos outcomes também. Quando mergearmos [248](https://github.com/Polkamarkets/polkamarkets-web/pull/248) a cor precisa ser pega com `getMarketColors ` e não mais por redux

```suggestion
  const market = useAppSelector(state => state.market.market);
  const colors = getMarketColors({
    network: market.network.id,
    market: market.id
  });
```